distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -234,19 +234,34 @@ chart:provider=default
 #####                   I/O component configurations                              #####
 #######################################################################################
 
-########################## Google Calendar configuration ##############################
+####################### Google Calendar (GCal) configuration ##########################
 #
-# the username and password for Google Calendar Account
-#gcal:username=
-#gcal:password=
+# Before using GCal, you need to have a Google API
+# Console project. The Wiki describes in detail the steps necessary to set 
+# up your Google API Console project, as well as how to obtain the credentials 
+# necessary to complete the information in this file. Once the project is created, 
+# and you've completed the steps described in the Wiki, you need to copy 
+# the "Client ID" and "Client secret" from the Credentials page on 
+# console.developers.google.com
+#
+# Copied from the "Client ID" field on the Credentials page (required)
+#gcal:client_id=
+
+# Copied from the "Client secret" field on the Credentials page (required)
+#gcal:client_secret=
 
-# the url of the calendar feed
-#gcal:url=
+# This is the name you gave to your Google Calendar.  GCal will download calendar 
+# events from this calendar (required)
+#gcal:calendar_name=
```

Comment:
The code says:
```
            String urlString = (String) config.get("calendar_name");
            if (!StringUtils.isBlank(urlString)) {
                calendar_name = urlString;
            } else {
                logger.error(
                        "gcal:calendar_name must be configured in openhab.cfg. Calendar name or word \"primary\"  MUST be specified");
                throw new ConfigurationException("calendar_name",
                        "gcal:calendar_name must be configured in openhab.cfg. Calendar name or word \"primary\"  MUST be specified");
            }
```
but the member `calendar_name` defaults to `"primary"` as well.  So the code might allow this to be optional.  For this PR, could you add that the special keyword `primary`" can be used?  Ideally it would be verified that specifying `calendar_name` is optional and defaults to `primary` and the code changed to allow it to be optional (and tested!). 😄